packages/server/src/unleash.ts

```diff
@@ -14,6 +14,7 @@ if (env.NODE_ENV === "production" && !isLocalhost()) {
 const defaultFeatures = {
     "dekoratoren.skjermdeling": true,
     "dekoratoren.chatbotscript": true,
+    "dekoratoren.umami": false,
 };
 
 // TODO: Features should be loaded on the client to avoid caching.
```

Comment:
Burde vi fikset dette nå?

Reply:
Jeg vet ikke hvordan isåfall, tenker kanskje uansett det burde løses i en annen oppgave så ikke denne vokser for mye?